### ISSUE TITLE
add weight to loss function

### ISSUE THREAD

**Issue by masoud-khalilian** [masoud-khalilian/mldl-waste#31]
In order to minimize the effect of class imbalance in our model.
The following link has a solution to class imbalance [ENet - Real-Time Semantic Segmentation](https://github.com/iArunava/ENet-Real-Time-Semantic-Segmentation/blob/master/ENet-Real_Time_Semantic_Segmentation.ipynb)

```
nn.CrossEntropyLoss(**weight=torch.FloatTensor(class_weights).to(device)**)
nn.CrossEntropyLoss().cuda() 
```
